### PR TITLE
redpen 1.3.0

### PR DESCRIPTION
**Pull request by takahi-i**
Updating the redpen brew formula to deploy the latest version (1.3.0).

### CODE REVIEW

File: Library/Formula/redpen.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Redpen < Formula
   desc "Proofreading tool to help writers of technical documentation"
   homepage "http://redpen.cc/"
-  url "https://github.com/recruit-tech/redpen/releases/download/v1.2.0/redpen-1.2.0.tar.gz"
-  sha1 "b1b49afd6a415572b5845a2c0e5144cffb65c4de"
+  url "https://github.com/recruit-tech/redpen/releases/download/v1.3.0/redpen-1.3.0.tar.gz"
+  sha1 "b4c740cfc09c850c3260f877aa0d4d10f26e1d68"
```
Comment (bfontaine) [inline Library/Formula/redpen.rb]: Please use a `sha256` here.

**Comment by takahi-i** [comment]
@bfontaine I force-pushed the update commit which replace sha1 to  sha256 and the value.

**Comment by bfontaine** [comment]
Thank you @takahi-i for your contribution, we appreciate it! :smile:

**Comment by takahi-i** [comment]
Thank you very much @bfontaine!

**Pull request closed without merging**